bin/setup

```diff
@@ -5,6 +5,9 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
+# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
+mkdir -p .git/safe
```

Comment:
Valid concern, I took what we have in our [`bin/setup` for Suspenders] which also is taken from our [shell snippets]

[shell snippets]: https://github.com/thoughtbot/shell-snippets/blob/master/generic/add-bin-to-path.sh
[`bin/setup` for suspenders]: https://github.com/thoughtbot/suspenders/blob/b3ee72c041d017c21bfc1970da20a9d36aed0f9f/templates/bin_setup#L16-L17